Tests/Fixtures/OHHTPStubs+Extension.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
```

Comment:
I think the name of this file could be more descriptive. Something like `+Paths` or `+Files` or maybe even `+Filesystem`.